src/RCDEF.CPP

```diff
@@ -695,7 +695,7 @@ int CDEC main( SI argc, char * argv[] )
 	FILE* fdtyph = NULL;
 	if (HFILESOUT)                      // not if not outputting .h files
 	{
-		sprintf( fdtyphname, "%s\\dtypes.hx",incdir);
```

Comment:
What does Windows do with `/` path delimiters? It's obviously passing, but ultimately it's probably best to use something wrapped around filesystem functionality that will handle path delimiters for the target file system.

Reply:
I couldn't find a way so filesystem decide the type of slash to use. I create a function that will handle it along with creating the path.